src/shared/components/layout/Footer/Footer.module.scss

```
@@ -46,7 +46,7 @@
     @include m.mq(md) {
       width: 100%;
       @include m.flex-box(space-evenly, center, row);
-      gap: 32px;
+      gap: 220px;
```

Stylelint (17.3.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)